server/issue_parser.go

```diff
@@ -28,6 +28,12 @@ func mdKeySummaryLink(issue *jira.Issue) string {
 	return fmt.Sprintf("[%s](%s%s)", issue.Key+": "+issue.Fields.Summary, issue.Self[:pos], "/browse/"+issue.Key)
 }
 
+func reporterSummary(issue *jira.Issue) string {
+	avatarLink := fmt.Sprintf("![avatar](%s =30x30)", issue.Fields.Reporter.AvatarUrls.One6X16)
```

Comment:
I'm wondering if we can assume that the avatar URL always exists here. I think Jira assigns a default if not set by the user.

Reply:
@mickmister I think jira doesn't have a default.